DESCRIPTION

```diff
@@ -8,7 +8,7 @@ Authors@R: c(
            email = "allyson.m.stoll@gmail.com"),
     person(given = "Kingslin",
            family = "Lv",
-           role = c("aut", "cre"),
+           role = c("aut"),
```

Comment:
Removing "cre" will cause `check()` to not pass. We need to have at least one person with "cre" role.

Reply:
I was removing as I got the ci-cd action running error; let me add it back to see if there is another error in the description file.